### ISSUE TITLE
Add support for C++

### ISSUE THREAD

**Issue by photex** [dzannotti/vscode-theme-spacemacs#11]
I'm happy to help out here. Just creating the issue for the sake of it.

I'll fork and see if I can't get a PR together.

Cheers!

**Comment by anthony-y** [comment]
Yep I would love this too. Never worked on a theme before but I'll clone and have a look around, see if I can help out
